channeldb/payment_status.go

```diff
@@ -8,17 +8,21 @@ const (
 	// and hence is unknown.
 	StatusUnknown PaymentStatus = 0
 
+	// StatusInitiated is the status where a payment has just been
+	// initiated.
+	StatusInitiated PaymentStatus = 1
```

Comment:
Can we really update these values without breaking anything? Isn't this payment status persisted in the payments DB? Or is this purely the in-memory representation?

Reply:
Ah, I see that this is returned from `decidePaymentStatus` which creates the status from boolean flags. Okay, that seems to work then. Maybe worth adding a sentence to the commit message?

Reply:
Updated the commit message to address it!